benchmark/engines/create-liquid.js

```diff
@@ -7,7 +7,7 @@ function createEngine (pkg) {
     cache: true,
     extname: '.liquid'
   })
-  liquid.registerFilter('url', path => `http://example.com${path}`)
+  liquid.registerFilter('url', path => `https://example.com${path}`)
```

Comment:
this includes `example.com` which is available via https (happy to drop `example.com` specifically if requested)